src/tests/fake/samples.tsx

```diff
@@ -0,0 +1,104 @@
+import { faker } from "@faker-js/faker";
+import { assign, pick } from "lodash";
+import nock from "nock";
+import { LabelNested } from "../../js/labels/types";
+import { LibraryType, SampleMinimal, WorkflowState } from "../../js/samples/types";
+import { createFakeLabelNested } from "./labels";
+import { createFakeUserNested } from "./user";
+
+type CreateFakeSampleMinimal = {
+    labels?: LabelNested[] | number[];
+    host?: string;
+    isolate?: string;
+};
+
+/**
+ * Create a fake sample minimal object
+ *
+ * @param overrides - optional properties for creating a sample minimal with specific values
+ */
+export function createFakeSampleMinimal(overrides?: CreateFakeSampleMinimal): SampleMinimal {
+    const defaultSampleMinimal = {
+        id: faker.random.alphaNumeric(8),
+        name: faker.random.word(),
+        created_at: faker.date.past().toISOString(),
+        host: faker.random.word(),
+        isolate: faker.random.word(),
+        labels: [createFakeLabelNested()],
+        library_type: LibraryType.normal,
+        notes: faker.random.word(),
+        nuvs: faker.datatype.boolean(),
+        pathoscope: faker.datatype.boolean(),
+        ready: true,
+        user: createFakeUserNested(),
+        workflows: { aodp: WorkflowState.INCOMPATIBLE, nuvs: WorkflowState.NONE, pathoscope: WorkflowState.NONE },
+    };
+
+    return assign(defaultSampleMinimal, overrides);
+}
+
+type CreateSampleType = {
+    name: string;
+    isolate: string;
+    host: string;
+    locale: string;
+    library_type: string;
+    files: string[];
+    labels: number[];
+    subtractions: string[];
+    group: string | null;
+};
+
+type CreateFakeSample = CreateFakeSampleMinimal & {
+    files?: string[];
+    subtractions?: string[];
+    locale?: string;
+    group?: number | string | null;
+};
+
+/**
+ * Create a fake object with the required data for creating a sample
+ *
+ * @param overrides - optional properties for creating a sample with specific values
+ */
+export function createFakeSample(overrides?: CreateFakeSample): CreateSampleType {
+    const { files, ...props } = overrides || {};
+    const sampleMinimal = pick(createFakeSampleMinimal(props), ["name", "isolate", "host", "library_type"]);
+    const defaultCreateSample = {
+        ...sampleMinimal,
+        locale: faker.random.word(),
+        subtractions: faker.datatype.number(),
+        files: files || [faker.datatype.number(), faker.datatype.number()],
+        labels: [],
+        group: null,
+    };
+
+    return assign(defaultCreateSample, overrides);
+}
```

Comment:
Assignment the docstring is on point this might be too specific for the mock file. The specific object for creating a sample should only be used in the sample creation view, so I am not sure the benefit of having to available to all tests.  If it is needed in multiple components happy to leave it here, but it should probably be renamed. Current naming scheme is exactly what I would expect a function that returns a complete `Sample` to return